test/io/ferguson/recordeof.good

```diff
@@ -0,0 +1 @@
+Done
```

Comment:
This test is a test for the specific EOF bug I fixed. There are other tests that write and read records.